ietf/bin/rfc-editor-index-updates

```diff
@@ -48,11 +46,18 @@ if options.skip_date:
 log("Updating document metadata from RFC index going back to %s, from %s" % (skip_date, settings.RFC_EDITOR_INDEX_URL))
 
 
-socket.setdefaulttimeout(30)
-rfc_index_xml = requests.get(settings.RFC_EDITOR_INDEX_URL).text
+response = requests.get(
+    settings.RFC_EDITOR_INDEX_URL,
+    timeout=30,  # seconds
+)
+rfc_index_xml = response.text
 index_data = ietf.sync.rfceditor.parse_index(io.StringIO(rfc_index_xml))
 
-errata_data = requests.get(settings.RFC_EDITOR_ERRATA_JSON_URL).json()
+response = requests.get(
+    settings.RFC_EDITOR_ERRATA_JSON_URL,
+    timeout=30,  # seconds
+)
+errata_data = response.json()
```

Comment:
The addition of the timeout opens the possibility of raising a requests.exceptions.Timeout exception. It would be nice to handle that, though it would be understandable if you decide that not erroring out of the command line utility is a correct behavior.